src/lib/__generated/graphql.schema.graphql

```diff
@@ -121,9 +123,10 @@ enum AssetOrder {
 }
 
 """
-To have author-related properties [See type definition](https://app.contentful.com/spaces/nvxmqufxogry/content_types/componentAuthor)
+To have author-related properties [See type definition](https://app.contentful.com/spaces/7b6c43hr18k6/content_types/componentAuthor)
```

Comment:
which space does this new spaceId correspond to?

Reply:
Probably my test space. Should we even be including the __generated directory? 🤔

Reply:
mm it's a good question. Based on the README it seems we expect the user to only `(re-)generate the GraphQL schema, types and sdk` if the content model changes, while we take care of the initial generation.

Reply:
I re-inserted the old space ID in the newly generated files even though they were created from a different space. I hope it works after merging.